src/api/flaskr/service/order/funs.py

```
@@ -364,115 +399,737 @@ def generate_charge(
+
+
+def handle_stripe_webhook(
+    app: Flask, raw_body: bytes, sig_header: str
+) -> Tuple[Dict[str, Any], int]:
+    provider = get_payment_provider("stripe")
+    try:
+        notification: PaymentNotificationResult = provider.handle_notification(
+            payload={"raw_body": raw_body, "sig_header": sig_header}, app=app
+        )
+    except Exception as exc:  # pragma: no cover - verified via tests for error path
+        app.logger.exception("Stripe webhook verification failed: %s", exc)
```

Ruff (0.14.4)

Redundant exception object included in `logging.exception` call

(TRY401)